sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs

```diff
@@ -76,13 +76,13 @@ public static void Verify(Compilation compilation, DiagnosticAnalyzer[] diagnost
             }
         }
 
-        private static void CompareActualToExpected(Compilation compilation, IEnumerable<Diagnostic> diagnostics, List<IIssueLocation> expectedIssues, bool compareIdToMessage)
+        private static void CompareActualToExpected(Compilation compilation, IEnumerable<Diagnostic> diagnostics, ICollection<IIssueLocation> expectedIssues, bool compareIdToMessage)
```

Comment:
This method needs only the language string and not the full compilation object so we should pass version instead.